examples/demo/otel-collector-config.yaml

```diff
@@ -12,6 +12,7 @@ exporters:
 
   zipkin:
     url: "http://zipkin-all-in-one:9411/api/v2/spans"
+    use_proto: yes
```

Comment:
Would it be more user-friendly to explicitly have a "format" option with "json" and "protobuf" as choices?

Reply:
My personal preference is simply `format`.

Reply:
Let's use `format`.

Reply:
I would say zipkin idiomatic is `serializer` (see https://github.com/openzipkin/zipkin-go/blob/master/reporter/serializer.go) and then valid types `json`, and `proto`. Ref to https://github.com/envoyproxy/envoy/pull/6985/files#diff-14b9269528c04650b999a5344ba11168R99-R103